pkg/virt-handler/non-root.go

```diff
@@ -135,3 +138,58 @@ func (d *VirtualMachineController) nonRootSetup(origVMI, vmi *v1.VirtualMachineI
 	}
 	return nil
 }
+
+func (d *VirtualMachineController) ensurePVCsOwnershipFromDVs(vmi *v1.VirtualMachineInstance) error {
```

Comment:
shouldn't this be `ensurePVCsOwnership`, a PVC can be referenced directly as a volumeSource or the pvc can be referenced as a dataVolumesource. basically, two ways of doing the exact same thing.

Reply:
Ownership of PVC is handled here https://github.com/kubevirt/kubevirt/blob/main/pkg/host-disk/host-disk.go#L192 . Therefore here we are trying to handle the specific case that is not covered otherwise.